src/styles/blocks/recommended.scss

```diff
@@ -0,0 +1,18 @@
+.recommended {
+  &__products {
+    @include pageGrid;
+    row-gap: 40px;
+  }
+
+  &__product {
+    grid-column: 1 / -1;
+
+    @include onTablet {
```

Comment:
Try using `grid-column: 1 / -1` instead of `grid-column: 2 / -2` here, you'll notice the difference.

Reply:
Well, I've noticed my comment is not showing the part of the code I wanted it to. I'm talking about the `grid-column` inside the `onTablet`.

Reply:
Got it, the images get bigger. Thanks